### DISCUSSION TITLE
Energy Dashboard

### DISCUSSION THREAD

**Discussion by stevendodd** [BottlecapDave/HomeAssistant-OctopusEnergy#152] [Q&A] [answered]
Hi,

I have half hourly electric meeting meter readings enabled and I can see the data coming through in half hour intervals in the previous period entity.

The energy dashboard only shows the data in a daily chunk, is this expected behaviour or should I see it broken down with more granularity?

**Comment by BottlecapDave** [comment, marked as answer]
This is a limitation of the Home Assistant energy dashboard. The dashboard is built for live data and doesn't provide a way to add historic data. I'm currently working on an Octopus Home Mini integration which can provide live data, but you'll need to apply for the device.
